apps/projects/test/Projects.test.js

```diff
@@ -872,6 +848,46 @@ contract('Projects App', accounts => {
             await app.removeRepo(repoId, { from: repoRemover })
           })
         })
+
+        it('work can be accepted multiple times', async () => {
+          const bountyId = (await app.getIssue(repoId, 1))[0].toString()
+          //console.log(bountyId)
+          await bountiesEvents.contribute(root, bountyId, 5, { value: 5 })
+          await bountiesEvents.fulfillBounty(root, bountyId, [root], 'test')
+          await bountiesEvents.fulfillBounty(bountyManager, bountyId, [bountyManager], 'test2', { from: bountyManager })
+          await app.reviewSubmission(
+            repoId,
+            1,
+            0,
+            true,
+            'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDl',
+            [9],
+            { from: bountyManager }
+          )
+
+          await app.reviewSubmission(
+            repoId,
+            1,
+            1,
+            true,
+            'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDl',
+            [6],
+            { from: bountyManager }
+          )
+          
+          const bounty1 = await app.getIssue(repoId, 1)
+          assert.deepEqual(
+            bounty1,
+            [ 
+              new web3.BigNumber(56),
+              new web3.BigNumber(15),
+              new web3.BigNumber(0),
+              '0xffffffffffffffffffffffffffffffffffffffff',
```

Comment:
We should use ANY_ADDR (line 16) constant in places like here, otherwise, we define it at all?